javascript-loops/main.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable prefer-const */
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+
+}
+
+console.log('getNumbersToTen() ', getNumbersToTen());
+
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+console.log('getEvenNumbersToTwenty()', getEvenNumbersToTwenty());
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = '';
+  while (count < times) {
```

Comment:
This should be `<=`. If you pass in `10` as the value for `times`, then the loop would only run 9 times, but we'd want it to run 10 times.